api-scripts/feesfines/README.md

```diff
@@ -0,0 +1,22 @@
+## Workflow steps:
+
```

Comment:
I see multiple .tsv and .txt files. Are they all pre-requisites to run the JMeter script? If so how are these getting used? What are the steps? Could you please update Readme with the steps?

Reply:
Will resolve in scope of https://github.com/folio-org/perf-testing/pull/43#discussion_r551130416